spec/configurator_spec.rb

```diff
@@ -229,5 +248,51 @@ def define_questions(questions_hash)
         /'foobar'.*test\.unknown_q.*#{configure_file_path}/
       )
     end
+
+    it 'loads default values' do
+
+
+      str_ans = "I am a little teapot!!"
+      erb_ans = '<%= I_am_an_erb_tag %>'
+
+      define_questions({
+        test: {
+          string_q: {
+            question: 'String?',
+            type: 'string',
+            default: str_ans
+          },
+          string_erb: {
+            question: 'Erb?',
+            default: erb_ans,
+          },
+          integer_q: {
+            question: 'Integer?',
+            type: 'integer',
+            default: 10
+          },
+          bool_true: {
+            question: 'True?',
+            type: 'boolean',
+            default: 'true'
+          },
+          bool_false: {
+            question: 'False?',
+            type: 'boolean',
+            default: false
+          }
+        }
+      })
+
+      configure_with_input("\n\n\n\n\nyes\nno\n")
```

Comment:
So turns out that `booleans` can't have defaults (or I am not doing it correctly). Their is a latter commit that fixes this. But will also fix it now